students/AY1617S2/tanWangLeng/TanWangLeng-Resume.md

```diff
@@ -33,15 +45,58 @@ Information Systems Intern
 *Responsibilities:*<br>
 
 * Providing advice regarding integration of Microsoft Office into business processes.
+* Studying usage issues pertaining to Microsoft Excel in the financial department, and proposing solutions to improve the workflow.
+
+## Project Experiences
+
+### Contributions to OSS Projects
+
+External Projects:
+* [GitHub's Electron](https://github.com/electron/electron) (Contributor)
+    * [Fix webpreferences not accepting numeric options #8542](https://github.com/electron/electron/pull/8542)
+    * [Support message, nameFieldLabel and showsTagField for macOS dialog #8556](https://github.com/electron/electron/pull/8556)
+
+Internal Projects (for CS3281&2: Thematic Systems Project I/II):
+* [se-edu](https://github.com/se-edu) (Contributor)
+    * Progress Report: [Link](https://github.com/nus-oss/cs3281-website/blob/master/students/AY1617S2/tanWangLeng/TanWangLeng-Progress.md)
+    * Worked on improving the Java desktop projects used by CS2103T Software Engineering teaching team, that will be used by students as the foundation of their graded project.
+    * [se-edu's learningresources](https://github.com/se-edu/learningresources/): Contributed to book chapters
+        * [Cross Site Scripting](https://github.com/se-edu/learningresources/pull/4)
+
+### Other Projects
+
+* CP3108B: Independent Work [Orbital]
+    * Archive Repository: [Link](https://github.com/yamgent/orbital-2016)
+    * Used the Ruby on Rails framework
+    * Wrote a prototype website to explore possible improvements that could be made for the interface of the NUS course bidding website used by undergraduates.
+
+
+* Major Project (Temasek Polytechnic)
+    * As a lead programmer, developed a top-down 3D game in Unity3D engine titled "Morality" with a team of 5 other fellow student developers.
+    * Targeted towards youths aged 10 - 16, with the aim of teaching them good moral values and habits.
```

Comment:
Not required for this module, but you may want to create a webpage with some screenshots/videos/details of the game and link from here.